### PR TITLE
Updated cql-to-elm instructions in README.md

### PR DESCRIPTION
**Pull request by sorliog**
Pull requests into cql-execution require the following.
Submitter and reviewer should ✔ when done.
For items that are not-applicable, mark "N/A" and ✔.

[CDS Connect](https://cds.ahrq.gov/cdsconnect) and [Bonnie](https://github.com/projecttacoma/bonnie) are the main users of this repository.
It is strongly recommended to include a person from each of those projects as a reviewer.

# Summary
Instructions for translating CQL to ELM in [README.md](https://github.com/cqframework/cql-execution/blob/master/README.md#to-execute-your-cql) are out of date with [clinical_quality_language](https://github.com/cqframework/clinical_quality_language). This update changes the documentation to use the ```cql-to-elm-cli``` executable instead of the ```cql-to-elm``` library.

**Submitter:**
- [x] This pull request describes why these changes were made
- [ N/A ] Code diff has been done and been reviewed (it does not contain: additional white space, not applicable code changes, debug statements, etc.)
- [ N/A ] Tests are included and test edge cases
- [ N/A ] Tests have been run locally and pass
- [ N/A ] Code coverage has not gone down and all code touched or added is covered.
- [ N/A ] Code passes lint and prettier (hint: use `npm run test:plus` to run tests, lint, and prettier)
- [ N/A ] All dependent libraries are appropriately updated or have a corresponding PR related to this change
- [ N/A ] `cql4browsers.js` built with `npm run build:browserify` if source changed.

**Reviewer:**

Name:
- [ ] Code is maintainable and reusable, reuses existing code and infrastructure where appropriate, and accomplishes the task’s purpose
- [ ] The tests appropriately test the new code, including edge cases
- [ ] You have tried to break the code

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -194,13 +194,13 @@ const psource = new cql.PatientSource([ {
...
-  .catch(err => {
-    console.error(err);
-  });
+        .exec(psource)
+        .then(result => {
+          console.log(JSON.stringify(result, undefined, 2));
+        })
+        .catch(err => {
+          console.error(err);
+        });
```
Comment (cmoesel) [inline README.md:203]: Indenting 8 spaces looks a bit odd to my eye. I don't think we follow that convention in our code for this project. Can you please revert to the original two-space indenting here? That will also be more consistent with the similar block of code around line 140 above.

File: README.md [hunk 2]
```diff
@@ -251,62 +251,62 @@ define AllFalse: false and false
...
+          "type" : "And",
+          "operand" : [
+            { "valueType" : "{urn:hl7-org:elm-types:r1}Boolean", "value" : "false", "type" : "Literal" },
+            { "valueType" : "{urn:hl7-org:elm-types:r1}Boolean", "value" : "false", "type" : "Literal" }
+          ]
+        }
+      }
       ]}
-   }
+  }
```
Comment (cmoesel) [inline README.md:309]: This indent, on the other hand, looks good. I'm not sure how/why we had a three-space indent before. Thanks for fixing it!

**Review by cmoesel — CHANGES_REQUESTED** [review]
Thanks for the contribution. Your changes around `cql-to-elm-cli` look good! I've requested one change related to how you reformatted the example that uses promises -- but once you fix that, I think we're good to go!

File: README.md (hunk 1 above)
Comment (sorliog) [reply README.md]: I have made the changes requested in the second commit.

**Comment by cmoesel** [comment]
Thanks, @sorliog. It looks like `npm audit` is failing CI, but since this is a documentation-only PR, I don't think that it's fair to expect you to address `npm audit` issues. So... I will merge. Thanks again!
